src/main/webapp/app/shared/additional-feedback/additional-feedback.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component, Input } from '@angular/core';
+import { Feedback } from 'app/entities/feedback.model';
+
+@Component({
+    selector: 'jhi-additional-feedback',
+    templateUrl: './additional-feedback.component.html',
+    styleUrls: ['./additional-feedback.component.scss'],
+})
+export class AdditionalFeedbackComponent {
+    @Input()
+    additionalFeedback: Feedback[];
+    @Input()
+    additional: boolean;
```

Comment:
This boolean is a bit confusing given that you already provided *addition* feedback in the other input. How about we call the first input just `feedback: Feedback[]`? Then the meaning of this boolean is a bit more obvious imo.

Reply:
👍